src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ModuleBuilderImpl.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Runtime.InteropServices;
+
+namespace System.Reflection.Emit
+{
+    internal sealed class ModuleBuilderImpl : ModuleBuilder
+    {
+        private readonly AssemblyBuilderImpl _assemblyBuilder;
+
+        #region Internal Data Members
+
+        internal Dictionary<Assembly, AssemblyReferenceHandle> _assemblyRefStore = new Dictionary<Assembly, AssemblyReferenceHandle>();
+        internal Dictionary<Type, TypeReferenceHandle> _typeRefStore = new Dictionary<Type, TypeReferenceHandle>();
+        internal List<TypeBuilderImpl> _typeDefStore = new List<TypeBuilderImpl>();
+        internal int _nextMethodDefRowId = 1;
+        internal int _nextFieldDefRowId = 1;
+        internal const string ManifestModuleName = "RefEmit_InMemoryManifestModule";
```

Comment:
This module name does not make sense for persisted assemblies. "InMemory" implies that the module is not persisted.

Reply:
Removing `InMemory` part